MultiQuestensions/src/Beatmaps/PreviewBeatmapPacket.cpp

```diff
@@ -1,12 +1,21 @@
 #include "Beatmaps/PreviewBeatmapPacket.hpp"
+#include "custom-types/shared/register.hpp"
+using namespace MultiplayerExtensions::Beatmaps;
```

Comment:
Consider using either this or the following namespace block for your methods, but not both in the same region. If this file would like to have using namespace... Below the method definitions, consider placing it below the definitions instead, if not using a `using` anyways.